Makefile

```
@@ -590,6 +590,7 @@ $(FOMANTIC_DEST): $(FOMANTIC_CONFIGS) package-lock.json | node_modules
 	rm -rf $(FOMANTIC_DEST_DIR)
 	cp web_src/fomantic/theme.config.less node_modules/fomantic-ui/src/theme.config
 	cp web_src/fomantic/_site/globals/* node_modules/fomantic-ui/src/_site/globals/
```

as I remember for proper overrides you will need views, elements, modules directories as well
```suggestion
	cp -r web_src/fomantic/_site/* node_modules/fomantic-ui/src/_site/
```

---

We only have `web_src/fomantic/_site/globals/site.variables` in the source directory, so the current glob matches everything we override, but this may be better for later.